editor/import/resource_importer_imagefont.cpp

```diff
@@ -94,6 +94,7 @@ Error ResourceImporterImageFont::import(const String &p_source_file, const Strin
 	Error err = ImageLoader::load_image(p_source_file, img);
 	ERR_FAIL_COND_V_MSG(err != OK, ERR_FILE_CANT_READ, vformat("Can't load font texture: \"%s\".", p_source_file));
 
+	ERR_FAIL_COND_V_MSG(columns <= 0 || rows <= 0, ERR_FILE_CANT_READ, vformat("Columns (%d) and rows (%d) cannot be zero.", columns, rows) );
```

Comment:
```suggestion
	ERR_FAIL_COND_V_MSG(columns <= 0, ERR_FILE_CANT_READ, vformat("Columns (%d) must be positive.", columns));
	ERR_FAIL_COND_V_MSG(rows <= 0, ERR_FILE_CANT_READ, vformat("Rows (%d) must be positive.", rows));
```
Clearer and more correct error message